modules/appeals_api/app/models/appeals_api/profile_veteran.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class ProfileVeteran
```

Comment:
This was an unused class. I renamed so that I could move ahead with my intended veteran class.

Reply:
I beleive it's used when determining a veteran's profile data for building their VANotify data to send them emails.

Reply:
You were totes right. I've refactored this into the current PR, which is now 5-bagillion lines long.